ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -107,50 +126,50 @@ def collect(self, device, unused):
 
         results['nova_url'] = yield nova.get_url()
 
-        result = yield keystone.tenants()
-        results['tenants'] = result.get('tenants', [])
+        _tenants = yield keystone.tenants()
```

Comment:
Something else we could potentially do at this level is provide a nerd knob to disable specific api calls / data collection.  (a zprop that lets you disable things seems to be a popular feature.. of course if we do it at the api call level, it might be possible for the users to really shoot themselves in the foot.. but it's also pretty powerful, so.. something to consider.)